megamek/src/megamek/common/LargeSupportTank.java

```diff
@@ -49,6 +50,15 @@
     // tanks have no critical slot limitations
     private static final int[] NUM_OF_SLOTS = { 25, 25, 25, 25, 25, 25, 25, 25 };
 
+    // maps ToHitData - SIDE_X constants to LOC_X constants here for hull down fixed side hit locations
```

Comment:
This should be in SuperHeavyTank, which is the parent class of LargeSupportTank, so it will cover both.

Reply:
Actually, nevermind about that. SuperHeavyTank overrides `rollHitLocation, so this never gets used anyway.

Reply:
To be honest, given SuperHeavyTank's "rollHitLocation" implementation, I think this discussion is moot - a hit on a particular side always directly translates to a single hit location with no random variation (unlike a standard-sized tank, for example, a side hit can't hit the front or the back). So even if a superheavy tank is considered hull down, it doesn't affect which location gets hit with an incoming attack. I'm going to remove the specialized location mapping I introduced, because it's already carried out within that method.

Reply:
That being said, I don't think the "hull down"/"partial cover" rules are being applied correctly to large vehicles currently anyway, I'm not seeing a partial cover/hull down modifier, and, as written, attacks over the partial cover won't actually hit the turret. Might as well see if I can get that implemented while I'm in there.

Reply:
I took a look at the relevant code and I'm not optimistic that I won't break something else while implementing that, so maybe I'll focus on just fixing the existing bug and hold off on dealing with superheavy tank hulldown/partial cover rules until after the dev release.